datastore/tests/unit/test_transaction.py

```diff
@@ -12,22 +12,55 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import mock
 import unittest
 
-import mock
+
+class TestTransactionOptions(unittest.TestCase):
+
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.datastore_v1.types import TransactionOptions
+        return TransactionOptions
+
+    def _make_one(self, **kw):
+        return self._get_target_class()(**kw)
+
+    def test_read_only(self):
+        options = self._make_one(read_only=self._get_target_class().ReadOnly())
+        self.assertTrue(options.HasField('read_only'))
+
+    def test_read_write(self):
+        options = self._make_one(read_write=
+                                 self._get_target_class().ReadWrite())
+        self.assertTrue(options.HasField('read_write'))
+
+    def test_previous_transaction(self):
+        previous_transaction = b'234'
+        options = self._make_one(
+                           read_write=self._get_target_class().ReadWrite(
+                               previous_transaction=previous_transaction))
```

Comment:
This method should be deleted? (Per my previous comment, this whole class should.)

----

Just for the sake of explanation: What rules is your editor using to make these indents? They don't seem to make sense? Please indent 4 spaces. I would **much** prefer to see:

```python
klass = self._get_target_class()
options = self._make_one(
    read_write=klass.ReadWrite(
        previous_transaction=previous_transaction,
    ),
)
```

Note also the call to `_get_target_class()` **before** it is used (there is such a thing as doing "too much" in a single expression).